CustomFeeValidator.cs

```diff
@@ -30,9 +32,14 @@ static bool Load(UnityModManager.ModEntry modEntry)
 
         static void OnGui(UnityModManager.ModEntry modEntry)
         {
-            loggingEnabled = GUILayout.Toggle(loggingEnabled, "enable logging");
+            settings.Draw(modEntry);
         }
 
+        static void OnSaveGui(UnityModManager.ModEntry modEntry)
+		{

```

Comment:
Indentation is messed up here. Make sure your editor is configured to use spaces for indentation, not tabs.

Reply:
✅ fixed